src/loader/mod.rs

```diff
@@ -513,8 +513,7 @@ impl KernelLoader for Image {
 #[cfg(test)]
 mod test {
     use super::*;
-    #[cfg(any(feature = "elf", feature = "bzimage"))]
-    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    #[cfg(any(feature = "elf", feature = "bzimage", feature = "image"))]
```

Comment:
These are all the possible features, so no need for the `cfg(any)`.

Reply:
ok. It's removed now.